scapy/layers/bluetooth.py

```diff
@@ -394,6 +416,12 @@ class EIR_Hdr(Packet):
         }),
     ]
 
+    def post_build(self, p, pay):
```

Comment:
* I don't think it is related to Windows... So maybe it should go in another PR.
* Cannot we use a `*LenField()` for the `len` field value and remove this method?

Reply:
Right. As the fix became smaller, I'll let it with this PR...